separate_proxy.yaml

```diff
@@ -0,0 +1,68 @@
+tosca_definitions_version: cloudify_dsl_1_0
+
+imports:
+  - snmp_types.yaml
+
+
+node_templates:
+
+  SNMPSecurityGroup:
+    type: SNMPSecurityGroup
+
+  SNMPMonitoredHost:
+    type: cloudify.openstack.nodes.Server
+    properties:
+      cloudify_agent:
+       user: ubuntu
+      server:
+        image: 9d25fe2d-cf31-4b05-8c58-f238ec78e633 # Ubuntu Server 14.04.1 LTS
+        flavor: 101
+    relationships:
+     - type: cloudify.openstack.server_connected_to_security_group
+       target: SNMPSecurityGroup
+
+  SnmpdConfiguringNode:
+    type: SnmpdConfiguringNode
+    relationships:
+      - type: cloudify.relationships.contained_in
+        target: SNMPMonitoredHost
+
+  ProxyServer:
+    type: cloudify.openstack.nodes.Server
+    properties:
+      cloudify_agent:
+        user: ubuntu
+      server:
+        image: 261844b3-479c-5446-a2c4-1ea95d53b668
+        flavor: 101
+    interfaces:
+      cloudify.interfaces.monitoring_agent:
+        install:
+          implementation: diamond.diamond_agent.tasks.install
+          inputs:
+            diamond_config:
+              interval: 1
+        start: diamond.diamond_agent.tasks.start
+        stop: diamond.diamond_agent.tasks.stop
+        uninstall: diamond.diamond_agent.tasks.uninstall
+
+  ProxyNode:
+    type: SNMPProxy
+    relationships:
+      - type: cloudify.relationships.contained_in
```

Comment:
should this one also have this:
`target: SnmpdConfiguringNode # Assure that Diamond starts after snmpd is configured`